### DISCUSSION TITLE
Help improve extension publishing experience!

### DISCUSSION THREAD

**Discussion by seaniyer** [microsoft/vscode-discussions#766] [Announcements]
Inviting all VS Code extension developers!

**Take this [publisher survey](https://www.surveymonkey.com/r/MMGFDBP?ref=mgh) and share your valuable feedback, please!**. It can help our teams deliver a better publishing experience for everyone. 

The VS Marketplace team is launching the survey to get perspectives on your publishing journey and what improvements would have the broadest and biggest benefits overall. Thanks for your time and for being part of the VS Marketplace community!

Sean 
Visual Studio Marketplace team

**Comment by zeluspudding** [comment]
Now that you mention it, I've got [this very specific extension publishing / distribution question](https://github.com/microsoft/vscode-discussions/discussions/768#discussion-5532096).
